src/ui/public/directives/kbn_top_nav.js

```diff
@@ -26,6 +26,8 @@ module.directive('kbnTopNav', function (Private) {
       label: _.capitalize(opt.key),
       hasFunction: !!opt.run,
       description: ('Toggle ' + opt.key),
+      noButton: !!opt.noButton,
+      openByDefault: !!opt.openByDefault ,
```

Comment:
How am I supposed to get access to kbnTopNav? It's only accessible within the scope of the `kbn-top-nav` element.